Shared/gta/VehicleUpgrades.h

```diff
@@ -0,0 +1,223 @@
+    {583, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {585, {1142, 1143, 1144, 1145, 1001, 1003, 1023, 1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082,
+           1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1087, 1006, 1018, 1019, 1020, 1013, 1086, 1007, 1017}},
+    {587, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {588, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {589, {1144, 1145, 1000, 1016, 1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085,
+           1096, 1097, 1098, 1008, 1009, 1010, 1087, 1004, 1005, 1006, 1018, 1020, 1013, 1024, 1086, 1007, 1017}},
+    {594, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {596, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {597, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {598, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {599, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {600, {1007, 1017, 1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085,
+           1096, 1097, 1098, 1008, 1009, 1010, 1087, 1004, 1005, 1006, 1018, 1020, 1022, 1013, 1086}},
+    {601, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {602, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {603, {1142, 1143, 1144, 1145, 1001, 1023, 1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083,
+           1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1087, 1006, 1018, 1019, 1020, 1024, 1086, 1007, 1017}},    
+    {604, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+    {605, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},    
+    {609, {1025, 1073, 1074, 1075, 1076, 1077, 1078, 1079, 1080, 1081, 1082, 1083, 1084, 1085, 1096, 1097, 1098, 1008, 1009, 1010, 1086, 1087}},
+};
+
+bool isUpgradeCompatibleWithModel(unsigned short model, unsigned short upgrade)
```

Comment:
As this function is within a header, it's better to make it inline. In addition, the function's name should be in UpperCamelCase.